packages/tasks/src/hardware.ts

```diff
@@ -168,6 +176,14 @@ export const SKUS = {
 				tflops: 9.22,
 				memory: [6],
 			},
+			"GTX 1080 Ti": {
+				tflops: 11.3,
```

Comment:
float32, no?

Reply:
@pcuenca My bad! I might have added fp32 instead, I am not sure what the rate of fp16 is so you can remove this card.